test/encoders.jl

```diff
@@ -21,6 +22,17 @@
     @test !haskey(map, 1)   # already encoded
 
     @test Xenc == MLJFlux.ordinal_encoder_fit_transform(X; featinds = [2, 3])[1]
+
+    # Test Consistency with Types
+    scs = schema(Xenc).scitypes
+    ts  = schema(Xenc).types
+    
+    # 1) all scitypes must be exactly Continuous
+    all(scs .== Continuous)  || error("Not all scitypes are Continuous")
```

Comment:
It's not usual to throw errors manually in tests. So:

```suggestion
    @test all(scs .== Continuous)
```

Reply:
Sorry I was testing it in another file without using Test for quickness and then hastly copy-pasted.